portality/static/js/doaj.fieldrender.edges.js

```diff
@@ -2320,7 +2320,20 @@ $.extend(true, doaj, {
 
                 var value = false;
                 if (ft === "terms" || ft === "term") {
-                    value = el.attr("data-value");
+                    val = el.attr("data-value");
```

Comment:
Do we need this because of inconsistencies in our index, e.g. storing `'false'` as a string vs `false` in `bibjson.copyright.author_retains`?

We've previously had a function for Yes / No depending on what's come from the index, that's handled at a different layer to this.

Reply:
The "Has Seal" filter works with "Yes" because we store the same information in two places: `admin.seal` with bool (`true/false`) and `index.has_seal` with string (`"Yes"/"No"`).
We only have `bibjson.preservation.has_preservation` field storing bool. We need to compare it to the string from the HTML data-value attr.

To treat the new filter the same as the existing one, we would need to add the field to `index` (?) that translates the bool value from bibjson to "Yes"/"No" and then uses this one. That, of course, requires migration. And later the same treatment for any new filter.